src/batch-client/index.ts

```diff
@@ -1017,9 +1019,7 @@ export class ZimbraBatchClient {
 			const sessionId = get(response, 'header.context.session.id');
 			const notifications = get(response, 'header.context.notify.0');
 
-			if (sessionId) {
-				this.sessionId = sessionId;
-			}
+			this.checkAndUpdateSessionId(sessionId);
```

Comment:
this function checks for sessionId === 1, in which response are we going to get 1 as session id?

Reply:
updated